SPECS/gcc/gcc.spec

```diff
@@ -329,6 +329,11 @@ LD=ld \
     --enable-plugin \
     --enable-shared \
     --enable-threads=posix \
+%ifarch x86_64
```

Comment:
Could we also add that to `config_cross_target`? Something along these lines:
```bash
    case $target in
	aarch64-*)	with_arch_arg="--with-arch=armv8.1-a";;
    esac
(..)
        --target=$target \
        "$with_arch_arg" \
        --with-ld=/usr/bin/$target-ld \
...
```

Reply:
We can, although I'd prefer to do that in a separate PR, since I've already tested as-is and am not very familiar yet with the cross-compilation here.